astroid/rebuilder.py

```diff
@@ -135,7 +136,7 @@ def _get_context(
             "ast.Tuple",
         ],
     ):  # TODO return type needs change to _Context enum
-        return self._parser_module.context_classes.get(type(node.ctx), astroid.Load)
+        return self._parser_module.context_classes.get(type(node.ctx), Load)
```

Comment:
I would just import `Context` and then use `Context.Load`. That would also make it a bit clearer.

Reply:
Yes I was trying to not change the way it was done before but it's hard to justify doing it like this.